client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigatedPersonInfo.tsx

```diff
@@ -83,9 +83,9 @@ const InvestigatedPersonInfo = (props: Props) => {
         return check ? yes : no;
     };
 
-    const validateStatusReason = async (newStatusReason : string | null) => {
```

Comment:
there is a reason for removing the async and await from this function?

Reply:
it's just not needed, validateSync is a synchronous function and async and await is redundant